praw/models/reddit/more.py

```diff
@@ -1,5 +1,5 @@
 """Provide the MoreComments class."""
-from typing import Any, Dict, List, TypeVar
+from typing import Any, Dict, List, TypeVar, Union
```

Comment:
This should have been caught in the initial review, but can you add a new line after this import, before the `...const` import?

Reply:
Sure.